agent/consul/state/catalog.go

```diff
@@ -57,11 +57,11 @@ func nodesTableSchema() *memdb.TableSchema {
 	}
 }
 
-// terminatingGatewayServicesTableSchema returns a new table schema used to store information
+//  gatewayServicesTableNameSchema returns a new table schema used to store information
 // about services associated with terminating gateways.
-func terminatingGatewayServicesTableSchema() *memdb.TableSchema {
+func gatewayServicesTableNameSchema() *memdb.TableSchema {
```

Comment:
Does this table schema need to be different in ent or are terminating and ingress gateways not namespaceable?

Reply:
It shouldn't need to be different, since the `ServiceIDIndex` accounts for `EnterpriseMeta`.